config/versions.yaml

```diff
@@ -15,12 +15,12 @@ default: "0.8.0"
 kabanero:
 - version: "0.8.0"
   related-versions: 
-    cli-services: "0.7.0"
+    cli-services: "0.8.0-rc.1"
     landing: "0.8.0-rc.1"
     events: "0.1.0"
-    collection-controller: "0.8.0"
-    stack-controller: "0.8.0"
-    admission-webhook: "0.8.0"
+    collection-controller: "0.8.0-rc.1"
```

Comment:
Going forward (in 0.9.0) we're not going to change the `version` of the three internal components for each alpha or rc release, we're just going to leave it as the final version number, and change the image tag instead.